src/core/Slider/Track.tsx

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { TrackDisplayMode } from './Slider';
+
+function shouldDisplaySegment(segmentIndex: number, mode: TrackDisplayMode) {
+  if ('odd-segments' === mode && 0 === (segmentIndex + 1) % 2) {
+    return true;
+  } else if ('even-segments' === mode && 0 === segmentIndex % 2) {
+    return true;
+  } else {
```

Comment:
No need else.. just return false.
But I assume it is not even reachable, right?

Reply:
Since TrackDisplayMode can be called with 'none' it is possible to reach.

Reply:
Oh yeah, right..
So just return false, no need else here...